kitsune/questions/config.py

```diff
@@ -140,37 +140,37 @@
                         # TODO: Just use the IA topics for this.
                         # See bug 979397
                         (
-                            "download-and-install",
+                            "install-and-update",
                             {
-                                "name": _lazy("Download, install and migration"),
-                                "topic": "download-and-install",
-                                "tags": ["download-and-install"],
+                                "name": _lazy("Install and Update"),
```

Comment:
I think the "update" should start with lowercase right?
```suggestion
                                "name": _lazy("Install and update"),
```